source/_integrations/bosch_shc.markdown

```diff
@@ -14,33 +14,36 @@ ha_platforms:
 ha_zeroconf: true
 ---
 
-The Bosch SHC integration allows you to integrate your [Bosch Smart Home Controller](https://www.bosch-smarthome.com) into Home Assistant.
+The Bosch SHC integration allows you to connect your [Bosch Smart Home Controller](https://www.bosch-smarthome.com) to Home Assistant in order to control and monitor your Bosch Smart Home devices.
 
 There is currently support for the following device types within Home Assistant:
 
 - [Binary Sensor](#binary-sensor)
+- [Cover](#cover)
```

Comment:
Please also update the `ha_category` and `ha_platforms` front matter above at the top of the file.

Reply:
👍 thanks for the hint. Done